.github/workflows/ci-e2e.yaml

```diff
@@ -0,0 +1,44 @@
+# Copyright Contributors to the L3AF Project.
+# SPDX-License-Identifier: Apache-2.0
+#
+# For documentation on the github environment, see
+# https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners
+#
+# For documentation on the syntax of this file, see
+# https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions
+
+name: CI E2E build
+on:
+  pull_request: {}
+  push:
+    branches:
+      - main
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Update and firewall stop
+        run: |
+          sudo apt update
+          sudo systemctl stop ufw
+          sudo apt install -y iproute2
+          sudo apt install git curl hey
+      - name: Checkout repository
+        uses: actions/checkout@f43a0e5ff2bd294095638e18286ca9a3d1956744
+      
+      - name: Prep
+        run: |
+          sudo cp -r /home/runner/work/l3afd/l3afd /root
+          sudo git clone -b E2E-TEST https://github.com/Atul-source/l3af-arch.git /root/l3af-arch
```

Comment:
```suggestion
          sudo git clone https://github.com/l3af-project/l3af-arch.git /root/l3af-arch
```

Reply:
+1

Reply:
I will do this changes after other PR on l3af-arch repository merged , otherwise this workflow will fail.